conf/params.distrib.xml

```diff
@@ -4,7 +4,10 @@
   <itop_url>https://localhost/iTop</itop_url>
   <itop_login>admin</itop_login>
   <itop_password>admin</itop_password>
-  
+  <itop_rest_token/>
```

Comment:
Maybe also remove `itop_rest_token` from sample config as it is not to be actively used?

Reply:
ok done!